src/elements/content-sidebar/activity-feed/common/activity-message/ActivityMessage.scss

```diff
@@ -23,6 +23,27 @@
     color: $bdl-box-blue;
 }
 
+.bcs-ActivityMessage-truncated {
+    /* webkit-line-clamp only works in combinatio with -webkit-box */
+    /* stylelint-disable-next-line value-no-vendor-prefix */
+    display: -webkit-box;
+    max-height: 140px;
```

Comment:
Should we store this px amount in a variable? Or make a note about how we got 140px exactly?